Jenkinsfile

```diff
@@ -174,7 +174,7 @@ stage('SonarQube analysis') {
     if (!(env.BRANCH_NAME).startsWith('dependabot/')) {
       def scannerHome = tool 'SonarQubeScanner';
       withSonarQubeEnv(installationName: 'SonarQubeServer') {
-        sh "${scannerHome}/bin/sonar-scanner -X -Dsonar.projectKey=couchbackup -Dsonar.branch.name=${env.BRANCH_NAME}"
+        sh "${scannerHome}/bin/sonar-scanner -X Dsonar.qualitygate.wait=true -Dsonar.projectKey=couchbackup -Dsonar.branch.name=${env.BRANCH_NAME}"
```

Comment:
are you missing a `-` (dash) before the `D` here?

Reply:
Thanks! I fixed it in https://github.com/cloudant/couchbackup/pull/560/commits/9348a78d27bf84d9991dcb1b54d4eee9b96e0b40